pset-iterators.js

```diff
@@ -66,6 +74,15 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+const findpresidentsNamedJames = presidents.filter(function (president) {
+  return president.president.indexOf("James") >= 0
+
+})
+
+console.log(findpresidentsNamedJames)
```

Comment:
@thetripler Great work here looking up the president whose first name is James. Good way to utilize the `indexOf` method with James and return if the index is greater than or equal to 0.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)